index.js

```diff
@@ -14,6 +15,12 @@ var PROXY = process.env.PROXY || '';
 var VERBOSE = process.env.VERBOSE || false;
 var trelloAPI = process.env.TRELLO_API || 'https://api.trello.com';
 var requiredLists = ['accepted', 'delivered', 'rejected', 'finished', 'current', 'backlog', 'icebox'];
+var labelColors = {
```

Comment:
These are the 4 "story types" from Pivotal, since they don't exist in Trello we thought of adding them as labels. We have them differ from real "labels" by adding a `#` before and assigning them a default color